arcane/src/arcane/utils/PlatformUtils.cc

```diff
@@ -525,6 +525,25 @@ getRealTimeNS()
 /*---------------------------------------------------------------------------*/
 /*---------------------------------------------------------------------------*/
 
+extern "C++" ARCANE_UTILS_EXPORT Int64 platform::
+getPageSize()
+{
+#if defined(ARCCORE_OS_WIN32)
+  SYSTEM_INFO si;
+  GetSystemInfo(&si);
+  return si.dwPageSize;
+#elif defined(ARCANE_OS_LINUX)
+  return ::sysconf(_SC_PAGESIZE);
+#else
```

Comment:
Not sure why the define names are not very consistent!

Reply:
Yes you are right. `ARCANE_OS_LINUX` is the same as `ARCCORE_OS_LINUX` but it is not consistent. I will fix it. Thanks